site/docs/plugins/session.md

```diff
@@ -684,7 +684,7 @@ bot.command("reset", (ctx) => {
 One may argue well that explicitly using `await` is preferable over assigning a promise to `ctx.session`, the point is that you _could_ do this if you like that style better for some reason.
 
 ::: tip Plugins That Need Sessions
-Plugin developers that make use of `ctx.session` should always allow users to pass `SessionFlavor | LazySessionFlavor` and hence support both modi.
+Plugin developers that make use of `ctx.session` should always allow users to pass `SessionFlavor | LazySessionFlavor` and hence support both mode.
```

Comment:
This change is incorrect. The plural form of modus is modi. Source: https://en.wiktionary.org/wiki/modus#Noun. However, it would probably be appropriate to use _mode_ rather than _modus_ as a base form, so we would end up using _modes_ here.

Reply:
Hmm… modi, T.I.L.